pypit/arparse.py

```diff
@@ -1002,15 +1002,14 @@ def reduce_flatfield_method(self, v):
         v : str
           value of the keyword argument given by the name of this function
         """
-        allowed = ['polyscan', 'bspline']
+        allowed = ['bspline']
```

Comment:
Should we raise a deprecation warning/error if they try, 
and an advisement to use the bspline instead?